tools_iuc.yaml

```diff
@@ -948,6 +948,10 @@ tools:
     owner: iuc
     tool_panel_section_label: Extract Features
 
+  - name: orfipy
+    owner: iuc
+    tool_panel_section_label: Extract Features
```

Comment:
isn't that annotation?

Reply:
It generates FASTA and/or BED files, so I'm not sure if annotation fits.